abaplint.json

```
@@ -177,6 +177,11 @@
     "method_parameter_names": false,
     "inline_data_old_versions": true,
     "local_variable_names": false,
-    "class_attribute_names": false
+    "class_attribute_names": false,
+    "commented_code": {
+      "exclude": [],
+      "severity": "Error",
+      "allowIncludeInFugr": true
+    }
```

```suggestion
    "commented_code": true
```

this will use the default configuration, a bit easier to read...